scripts/migrate_project_mailing_lists.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+"""Create mailing lists for all top-level projects
+"""
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+
+from website.app import init_app
+from website.mailing_list.utils import create_list as real_create_list
+from website.models import Node
+from website.notifications.model import NotificationSubscription
+from website.notifications.utils import to_subscription_key
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+EVENT = 'mailing_list_events'
+SUBSCRIPTION_TYPE = 'email_transactional'
+
+def fake(*args, **kwargs):
+    # Override actual method to prevent outgoing calls
+    return
+
+def get_targets():
+    return Node.find()
```

Comment:
Also, there is now a predefined query for this sort of thing:

https://github.com/abought/osf.io/blob/7f1d5c78f8811141876e8e42fb30d56ba9b67bbc/website/project/utils.py#L24-L24

In particular, it seems like this definition would create mailing lists for bookmarks, collections, registrations, and deleted nodes- is that intentional?

```
>>> Node.find(utils.TOP_LEVEL_PROJECT_QUERY).count()
3035
```